sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerErrorCode.java

```diff
@@ -3,14 +3,26 @@
 /**
  * This enum provides exit codes for Schedulers.
  */
-public enum SchedulerErrorCode {
-    SUCCESS,
-    INITIALIZATION_FAILURE,
-    REGISTRATION_FAILURE,
-    RE_REGISTRATION,
-    OFFER_RESCINDED,
-    DISCONNECTED,
-    ERROR,
-    PLAN_CREATE_FAILURE,
-    LOCK_UNAVAILABLE
+public class SchedulerErrorCode {
```

Comment:
Avoids surprise instability when values are added/removed/reordered. A similar use of enums in cassandra led to bugs.

In this case I was removing the REGISTRATION_FAILURE enum as it's no longer being used, and I didn't want that to cause other values to change their ordinals.